packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1018,8 +1018,8 @@ errors.E0000150 = You have reached the limit of sms requests, please try again l
 errors.E0000151 = You have reached the limit of call requests, please try again later.
 
 # Consent
-oie.consent.admin.title = would like to access:
```

Comment:
@thomasmaslen-okta : Is it possible to re-write this string to include clarity on the subject and object in this string for translation accuracy and completeness. Currently this string is concatenated.

#Helper_Text: {0} refers to the name of an app. This string is followed by access options such as view email, phone number, etc.
oie.consent.admin.title= {0} would like to access:

Reply:
@thomasmaslen-okta - This should be possible by using our [HTML tag syntax](https://github.com/okta/okta-ui/#handlebars-i18n) in the template.

Let's restructure the `titleText` logic to use this new approach:

```properties
oie.consent.scopes.admin.title = <$1>{0}</$1> would like to access:
```

```js
// Inside of AdminConsentViewHeader
...
<span class="title-text">
  {{i18n code="oie.consent.scopes.admin.title" bundle="login" $1="<b>$1</b>"}}
</span>
```

This has a few additional benefits:
- No longer need to pass in the `no-translate` class, as it's processed as an argument inside of our i18n resource file
- We won't need to request additional translations if our markup changes

Reply:
@evelynteo-okta @jmelberg-okta I agree that this looks to me like a good way to handle this.

... but I will defer to the group who owns the affected area (Consent for OAuth scopes) and introduced these two properties in February in #1637 (cc @sherizada-okta) and may have had their reasons for doing it that way.

I am in a different group that inadvertently broke one of those OAuth-related strings in April when we added support for Email magic-link Consent in #1791, and in this PR I am just trying to undo that breakage -- I'm trying to restore the end-user OAuth Consent view to displaying the text that it ought to display, even if under the hood the way that we go about displaying that text isn't ideal.

In short, can we address such restructuring in a separate JIRA (preferably not with an OIE Tuscany deadline)?

Reply:
Seems reasonable to me, but this will produce **invalid translations** for non-English users.

@pradeepdewda-okta and @sherizada-okta - Should I create a JIRA for your team to address this before Tuscany? It should be a simple fix.

Reply:
> [...] this will produce **invalid translations** for non-English users.

I expect that is so (though I am eminently unqualified to judge), but FWIW whatever these translations _will_ do is whatever they have been doing since #1775 added translations for these in March (well, until #1791 added its infelicity in April).